news/698-bugfix.md

```diff
@@ -0,0 +1 @@
+Add logging of query statements to QueryToExecute for exceptions
```

Comment:
Maybe: 

CohortExtractor database queries that crash during execution are now logged

Reply:
Sure - updated.